x-pack/plugins/alerting/server/saved_objects/migrations.ts

```diff
@@ -25,6 +25,7 @@ import { extractRefsFromGeoContainmentAlert } from './geo_containment/migrations
 const SIEM_APP_ID = 'securitySolution';
 const SIEM_SERVER_APP_ID = 'siem';
 export const LEGACY_LAST_MODIFIED_VERSION = 'pre-7.10.0';
+const legacyThreatIndicatorPath = 'threatintel.indicator';
```

Comment:
Nit: As pointed out to me in the original 8.0 migration PR, better not to use `legacy` in the name, as there could be additional "legacy" fields in the future and it will be unclear which one we're referring to. We should get out of that habit, but this is something we can change later.

Reply:
updated variable name to `filebeat7xIndicatorPath` to be more explicit